sonic-utilities-tests/config_mgmt_test.py

```diff
@@ -0,0 +1,475 @@
+        cmdpb = self.config_mgmt_dpb(iConfig)
+        # create ARGS
+        dPorts, aPorts, pJson = self.generate_args(portIdx=8, laneIdx=73, \
+            curMode='2x50G', newMode='1x100G')
+        deps, ret = cmdpb.breakOutPort(delPorts=dPorts, addPorts= aPorts, portJson=pJson,
+            force=False, loadDefConfig=False)
+        # Expected Result
+        assert ret == False and len(deps) == 3
+        assert cmdpb.writeConfigDB.call_count == 0
+        return
+
+    def dpb_port8_4x25G_2x50G_f_l(self, iConfig):
+        cmdpb = self.config_mgmt_dpb(iConfig)
+        # create ARGS
+        dPorts, aPorts, pJson = self.generate_args(portIdx=8, laneIdx=73, \
+            curMode='4x25G', newMode='2x50G')
+        cmdpb.breakOutPort(delPorts=dPorts, addPorts= aPorts, portJson=pJson,
+            force=True, loadDefConfig=True)
+        # Expected Result dConfig and aConfig is pushed in order
+        dConfig = {u'ACL_TABLE': {u'NO-NSW-PACL-V4': \
+        {u'ports': ['Ethernet0', 'Ethernet4']}, u'NO-NSW-PACL-TEST': {u'ports': None}}, \
+        u'INTERFACE': None, u'VLAN_MEMBER': {u'Vlan100|Ethernet8': None, \
+        u'Vlan100|Ethernet11': None}, u'PORT': {u'Ethernet8': None, \
+        u'Ethernet9': None, u'Ethernet10': None, u'Ethernet11': None}}
+        aConfig = {u'ACL_TABLE': {u'NO-NSW-PACL-V4': \
+        {u'ports': ['Ethernet0', 'Ethernet4', 'Ethernet8', 'Ethernet10']}}, \
+        u'VLAN_MEMBER': {u'Vlan100|Ethernet8': {u'tagging_mode': u'untagged'}}, \
+        u'PORT': {'Ethernet8': {'speed': '50000', 'lanes': '73,74'}, \
+        'Ethernet10': {'speed': '50000', 'lanes': '75,76'}}}
+        assert cmdpb.writeConfigDB.call_count == 2
+        self.checkResult(cmdpb, dConfig, aConfig)
+        self.postUpdateConfig(iConfig, dConfig, aConfig)
+        return
+
+    def dpb_port4_4x25G_2x50G_f_l(self, iConfig):
+        cmdpb = self.config_mgmt_dpb(iConfig)
+        # create ARGS
+        dPorts, aPorts, pJson = self.generate_args(portIdx=4, laneIdx=69, \
+            curMode='4x25G', newMode='2x50G')
+        cmdpb.breakOutPort(delPorts=dPorts, addPorts= aPorts, portJson=pJson,
+            force=True, loadDefConfig=True)
+        # Expected Result dConfig and aConfig is pushed in order
+        dConfig = {u'ACL_TABLE': {u'NO-NSW-PACL-V4': \
+        {u'ports': ['Ethernet0', 'Ethernet8', 'Ethernet10']}}, u'PORT': \
+        {u'Ethernet4': None, u'Ethernet5': None, u'Ethernet6': None, \
+        u'Ethernet7': None}}
+        aConfig = {u'ACL_TABLE': {u'NO-NSW-PACL-V4': \
+        {u'ports': ['Ethernet0', 'Ethernet8', 'Ethernet10', 'Ethernet4']}}, \
+        u'PORT': {'Ethernet4': {'speed': '50000', 'lanes': '69,70'}, \
+        'Ethernet6': {'speed': '50000', 'lanes': '71,72'}}}
+        self.checkResult(cmdpb, dConfig, aConfig)
+        self.postUpdateConfig(iConfig, dConfig, aConfig)
+        return
+
+###########GLOBAL Configs#####################################
+configDbJson =  {
```

Comment:
I was thinking if you could create a folder containing two mock data json files like configDbJson and portBreakOutConfigDbJson and then import it to this function. 
something like below.
[This Function](https://github.com/Azure/sonic-utilities/blob/master/sonic-utilities-tests/acl_loader_test.py#L17) in acl_loader_test.py file uses [acl_input](https://github.com/Azure/sonic-utilities/tree/master/sonic-utilities-tests/acl_input) folder.
It will help to structurize this test by dividing the code and the input data. What do you think?

Reply:
Since right now file is relatively small, It seems okie to have json in file itself.